content/SCALE/SCALEUIReference/Credentials/LocalUsersScreensSCALE.md

```diff
@@ -83,35 +83,36 @@ Built-in users (except the **root** user) do not include the **Home Directory Pe
+{{< trueimage src="/images/SCALE/23.10/AddUserHomeDirPermSCALE.png" alt="Directories and Permissions Settings" id="7: Directories and Permissions Settings" >}}
 
 {{< truetable >}}
 | Setting | Description |
 |---------|-------------|
 | **Home Directory** | Enter or browse to enter the path to the home directory for this user. If the directory exists and matches the **Username**, it is set as the home directory for the user. When the path does not end with a subdirectory matching the username, a new subdirectory is created if **Create Home Directory** is selected (enabled). The full path to the user home directory displays in this field on the **Edit User** screen for this user. |  
-| **Home Directory Permissions** | Select the permissions checkboxes (**Read**, **Write**, **Execute**) for each (**User**, **Group**, **Other**) to set default Unix permissions for the user home directory. Built-in users are read-only and do not see these permissions settings.|
+| **Home Directory Permissions** | Select the permissions checkboxes (**Read**, **Write**, **Execute**) for each (**User**, **Group**, **Other**) to set Unix permissions for the user home directory. Built-in users are read-only and can not modify these permissions settings.|
 | **Create Home Directory** | Select to create a home directory for the user when the home directory path for this user does not end in the user name. Creates a home directory for the user within the selected path. |
 {{< /truetable >}}
 
 {{< /expand >}}
+
 ### Authentication settings
-**Authentication** settings specify authentication methods, the public SSH key, user administration access, and enables/disables password authentication. 
+**Authentication** settings specify authentication methods, the public SSH key, user administration access, and enables/disables password authentication.
 It also includes the **Shell** screen options.
 {{< expand "Click Here for More Information" "v" >}}
 
-{{< trueimage src="/images/SCALE/22.12/AddUserHomeDirAuthSCALE.png" alt="Add User Home Directory and Authentication Settings" id="8: Add User Home Directory and Authentication Settings" >}}
+{{< trueimage src="/images/SCALE/23.10/AddUserHomeDirAuthSCALE.png" alt="Authentication Settings" id="8: Authentication Settings" >}}
 
 {{< truetable >}}
 | Setting | Description |
 |---------|-------------|
-| **Authorized Keys** | Enter or paste the downloaded SSH *public* key of the user for any key-based authentication. Use **Download Authorized Keys** to obtain a public key text file. Keep a backup copy of the public key! Do not paste the private key in this field! | 
+| **Authorized Keys** | Enter or paste the downloaded SSH *public* key of the user for any key-based authentication. Use **Download Authorized Keys** to obtain a public key text file. Keep a backup copy of the public key! Do not paste the private key in this field! |
```

Comment:
I think *public* was legacy content before we had our style guide so we don't need to use italics for public.
```suggestion
| **Authorized Keys** | Enter or paste the downloaded SSH public key of the user for any key-based authentication. Use **Download Authorized Keys** to obtain a public key text file. Keep a backup copy of the public key! Do not paste the private key in this field! |
```